caps/capabilities_test.go

```diff
@@ -46,9 +46,11 @@ func (s *CapabilitySuite) TestValidateName(c *C) {
 
 func (s *CapabilitySuite) TestAdd(c *C) {
 	repo := NewRepository()
+	err := LoadBuiltInTypes(repo)
```

Comment:
I did exactly that but it's still a few more branches away. It's after the split to separate files.